sympy/solvers/solveset.py

```diff
@@ -1511,7 +1511,7 @@ def add_type(lhs, rhs, x):
 
     def add_type(lhs, rhs, symbol, domain):
         """
-        Helper for `_transolve` to handle equations of
+        Helper for ``_transolve`` to handle equations of
         `Add` type, i.e. equations taking the form as
         `a*f(x) + b*g(x) + .... = c`.
```

Comment:
This math should be fixed. Also `Add` above.

Reply:
This line won't be seen on the website. I guess this form would be readable for one who reads the code.

Reply:
Then it should use double backticks to indicate that it is code. 

Eventually we will fix it so that single backticks are code, but that will be a lot of work.

Reply:
https://github.com/sympy/sympy/issues/13519